lib/Conversion/TorchToTcp/Utils.cpp

```diff
@@ -529,6 +529,29 @@ void TorchToTcpCustomOpConversionHelper::addFloatAttr(std::string attrName,
       rewriter.getNamedAttr(attrName, rewriter.getF64FloatAttr(constVal)));
 }
 
+bool TorchToTcpCustomOpConversionHelper::tryConvertConstToFloatAttr(
+    std::string attrName, Value value) {
+  if (conversionResult.failed())
+    return false;
+
+  double constFPVal;
+  if (matchPattern(value, torch::Torch::m_TorchConstantFloat(&constFPVal))) {
+    attrs.push_back(
+        rewriter.getNamedAttr(attrName, rewriter.getF64FloatAttr(constFPVal)));
+    return true;
+  }
+
+  // convert constant int to fp if possible
+  int64_t constIntVal;
+  if (matchPattern(value, torch::Torch::m_TorchConstantInt(&constIntVal))) {
+    attrs.push_back(rewriter.getNamedAttr(
+        attrName, rewriter.getF64FloatAttr(static_cast<double>(constIntVal))));
+    return true;
+  }
```

Comment:
Why do we have to convert int to fp here? Why not convert them to `IntAttr`?

Reply:
`aten.arange.start_step` can take either `int` or `fp` for start and step arguments. However, trt-mlir only accepts [F64Attr](https://sourcegraph.robot.car/github.robot.car/cruise/mla-robocomp-tensorrt-mlir/-/blob/include/mlir-tensorrt/Dialect/TensorRT/IR/TensorRTOps.td?L786-787). Converting all constants to FPAttr saves us some time for type checking.

I can also add the conversion in tcp-to-trtmlir. LMKWYT

Reply:
Thanks for the clarification. Can you add a comment in the code to explain this?